systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthAuthorizationST.java

```diff
@@ -161,15 +159,81 @@ void testTeamAWriteToTopicStartingWithXAndTeamBReadFromTopicStartingWithX() thro
 
         KafkaTopicResource.topic(CLUSTER_NAME, topicName).done();
 
-        Future<Integer> producer = teamAOauthKafkaClient.sendMessagesTls(topicName, NAMESPACE, CLUSTER_NAME, TEAM_A_CLIENT,
-                MESSAGE_COUNT, "SSL");
+        Future<Integer> producer = teamAOauthKafkaClient.sendMessages(topicName, NAMESPACE, CLUSTER_NAME,
+                MESSAGE_COUNT);
+
+        assertThat(producer.get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS), is(MESSAGE_COUNT));
+
+        Future<Integer> consumer = teamBOauthKafkaClient.receiveMessages(topicName, NAMESPACE, CLUSTER_NAME,
+                MESSAGE_COUNT, "x_consumer_group_b");
+
+        assertThat(consumer.get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS), is(MESSAGE_COUNT));
+    }
+
+    @Description("As a superuser of team A and team B, i am able to break defined authorization rules")
+    @Test
+    void testSuperUserWithOauthAuthorization() throws InterruptedException, ExecutionException, TimeoutException, IOException {
```

Comment:
I think this tests the super users well. But this test it self doesn't contain any verifications that the authorization alone works. Do you think it is sufficient that it is coverd in other tests? Or should we have something like:
1. Setup the clusetr without super users
2. Test that they cannot read and write
3 Modify the cluster to add super users
4. Test that they can now read and write.

Reply:
I can add another test case for clarity with that scenario.

Reply:
I think it would be good, because right now here you do not really know whether the authorization works at all.

Reply:
I have decided to make it really simple and did these scenario in one test. It will be really time consuming if i do two separate test cases...